pynest/nest/raster_plot.py

```diff
@@ -106,21 +101,54 @@ def from_file(fname, **kwargs):
 
     Parameters
     ----------
-    fname : str
-        Name of file
+    fname : str or tuple(str) or list(str)
+        File name or list of file names
```

Comment:
I think it would be good to document how lists of files are handled. If I understand it right, data from files are concatenated and plotted as if they had been stored in a single file?